app/controllers/import_controller.rb

```diff
@@ -382,6 +386,33 @@ def validate_workflow_specific_enums(workflow, field, value, row_ind)
     end
     raise ValidationError.new(err_msg, row_ind)
   end
+
+  def validate_continuous_exposure(workflow, field, continuous_exposure_value, row_ind, last_date_of_exposure_value)
+    return false if workflow == :isolation || continuous_exposure_value.blank?
+
+    continuous_exposure_boolean = import_and_validate_bool_field(field, continuous_exposure_value, row_ind)
+
+    if continuous_exposure_boolean
+      @warnings[VALIDATION[field][:label]] = 'Your import contains one or multiple monitorees with Continuous Exposure ' \
```

Comment:
PO team has changed their mind on the wording: https://mitrecorp.slack.com/archives/C029JM3PFC4/p1638889561150700?thread_ts=1638888992.148800&cid=C029JM3PFC4

Reply:
@timwongj Updated based on new wording. Thank yoU!